### ISSUE TITLE
D1S1 - Intro to R

### ISSUE THREAD

**Issue by ndphillips** [therbootcamp/Erfurt_2018June#3]
### Slides

- Maybe show a few slides showing really cool outputs (plots, statistical analysis) with only a few lines of code. This emphasises that R has packages that make it really easy to do cool stuff with only a few lines. For example, after the tidyverse slide. 

- Could emphasise reproducibility as a benefit of R over other (paid) languages. 

- Could use more practical / applied examples. For example, instead of `some_name <- c(1, 2, 3)`, maybe use a practical example like `ages <- c(18, 54, 23)`

- Do we need to show `typeof()`? Or just use `class()`? Consider removing `typeof()`

- Getting pretty technical by slide 8. Some more applied examples might help.

- Could have a slide introducing packages. What is a package? What does it contain? How do you install a package? What does it mean to load a package? How do you know what the package contains? Emphasise that packages are always free, and are available either through CRAN or GitHub.

- Don't apologise for poorly wrapped error messages in slides :)

- In Slide 1-, when introducing `mean()` include the argument name `mean(x = c(1, 2, 3))` instead of `mean(c(1, 2, 3))`, or at least specify the difference and explain why we should always be explicit.

- Emphasise the importance of examples in help files.

- Occasionally getting perhaps a bit too technical (e.g.; explaining what methods mean in the help file).

- Show one nice well structured R script starting with loading packages, loading data, etc etc. Take people through the structure of what a real R script will look like.

- Have a slide showing what a nice workflow is. Create a project, create folders, put data in the folders (etc). 

- In short style guide, it's a bit awkward to have a name of an R script `analyze_baselers.R` in a code block.

### Practical

- Maybe two separate practical work flows. One `playground.R` script, where you create simple objects, and one called `analysis.R` where you create a script with a reasonable workflow (load packages, load data, etc. etc.). Give them an idea of how a good script will work

- Consider more applied examples instead of `my_vec`, `my_logical` (etc).  Instead create vectors representing age, gender, income. Create a tibble from these data. Then access specific columns
